bin/yaml/ygen.yaml

```diff
@@ -0,0 +1,16 @@
+compilers:
+  ygen:
+    ylc:
+      type: ziparchive
+      fetch: https://github.com/Cr0a3/ygen/archive/refs/heads/main.zip
+      script: |
+        cd ygen-main
+        sudo apt install rust -y
```

Comment:
Hi!

Sorry - but we can't do this: the system has no `sudo` and installing `rust` like this won't work across all our systems.

I replied on the feature request you opened (and then closed) - you'll probably need us to help and/or work with you to add support for building rust-based compilers like we build all our other compilers (C, C++, python, Rust itself, etc) with our `misc-builder` or `clang-builder` or `***-builder` repositories.

Just copying the binary here might not work long-term: this doesn't copy any DLLs it may need too. We can help with this though!

Reply:
Oh sorry, I did not think of this.
Luckily ylc does not depend on any dlls cuz it uses my own written code generation library.

Another way for builds is I could add a GitHub workflow which would build the toolchain in my main repo, release it with the commit id, which would then be downloadable (or just create a pr to misc-builder)

Reply:
A PR to misc-builder means we would do the CI ourselves with infrastructure that "guarantees" it will run. But also an installable binary for an old Ubuntu would work too, or even better statically linked against MUSL to make it truly standalone.

Thanks!

Reply:
I changched it, so now it should download my prebuild